src/search_unused.rs

```diff
@@ -163,10 +148,9 @@ fn collect_paths(dir_path: &Path, analysis: &PackageAnalysis) -> Vec<PathBuf> {
             // Remove .rs extension.
             path_buf.pop();
             root_paths.insert(path_buf);
```

Comment:
I think we could even construct `root_paths` from the iterator here, if it returned `path_buf`. That way, `root_paths` wouldn't need to be `mut`able anymore!